greedybear/settings.py

```diff
@@ -69,6 +75,35 @@
 
 WSGI_APPLICATION = "greedybear.wsgi.application"
 
+
+REST_FRAMEWORK = {
+    "DEFAULT_RENDERER_CLASSES": [
+        "rest_framework.renderers.JSONRenderer",
+    ],
+    "DEFAULT_AUTHENTICATION_CLASSES": [
+        "durin.auth.CachedTokenAuthentication",
+    ],
+    "DEFAULT_PERMISSION_CLASSES": ["rest_framework.permissions.IsAuthenticated"],
+    "EXCEPTION_HANDLER": "rest_framework.views.exception_handler",
+    "DEFAULT_SCHEMA_CLASS": "drf_spectacular.openapi.AutoSchema",
+}
+
+# DRF Spectacular
+SPECTACULAR_SETTINGS = {
+    "TITLE": "GreedyBear API specification",
+    "VERSION": "0.1.3",
```

Comment:
this version would be difficult yo keep updated. Would be better to keep it in a separate file and then write in the release template (see how we do it in IntelOwl [here](https://github.com/intelowlproject/IntelOwl/blob/master/.github/release_template.md) to remember to update it before each new release

Reply:
I am not sure what would be some other great place to keep that version , so rather placing it in a seperate file I left it there and referenced it `release_template.md` as it is done in Intelowl.

Reply:
a simple `version.py` file so the next time we would need to use that variable in other parts of the code we won't forget that we have already added a version in this file too